hazelcast/src/test/java/com/hazelcast/dataconnection/impl/ListResourcesTest.java

```diff
@@ -79,6 +82,16 @@ public static Object[] params() {
     @Parameter(1)
     public DataConnectionResource[] expectedResources;
 
+    @Before
+    public void setUp() {
+        if (OsHelper.isWindows()) {
```

Comment:
Why don't we rather use `DockerTestUtil.dockerEnabled()` method?

Reply:
You mean something like this ? 
```
@Before
public void setUp() {
  // If Windows
  if (!DockerTestUtil.dockerEnabled()) {
    assumeThat(provider)
                    .isInstanceOfAny(
                            H2DatabaseProvider.class
      );
   }
 }
````

Reply:
Here is a more generic approach
rev : f52cd34ac3ca97f420580adc96a861933039a44b